src/openrct2-ui/windows/SavePrompt.cpp

```diff
@@ -141,28 +170,22 @@ class SavePromptWindow final : public rct_window
                 return;
             case WIDX_CLOSE:
             case WIDX_CANCEL:
-                window_close(*this);
+                Close();
                 return;
         }
     }
 
     void OnDraw(rct_drawpixelinfo& dpi) override
     {
-        WindowDrawWidgets(*this, &dpi);
+        DrawWidgets(dpi);
     }
 };
 
-/**
- *
- *  rct2: 0x0066DCBE
- */
 rct_window* WindowSavePromptOpen()
 {
     int32_t width, height;
```

Comment:
Please move these to justbefore the if  that defines them. Also break to one per line and initialize them to a default value.